cmd/minikube/cmd/start.go

```diff
@@ -112,6 +115,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	api, err := machine.NewAPIClient()
+	ensureNotRunning(api, constants.DefaultMachineName)
```

Comment:
What happens if the already running instance has a different name? (like via the `--profile` argument to `minikube start`)

Reply:
nice catch, I will make the changes. Thanks